### PR TITLE
[RVW] Migrate Turing cert-manager to using ingressShim config instead of annotations

### PR DESCRIPTION
**Pull request by sgibson91**
This change is at the advice of @consideRatio, see https://github.com/jupyterhub/mybinder.org-deploy/issues/1148#issuecomment-588651294

### CODE REVIEW

File: config/turing.yaml [hunk 1]
```diff
@@ -212,3 +207,11 @@ gcsProxy:
   buckets:
     - name: mybinder-staging-events-archive
       host: archive-analytics-staging-mybinder.turing.10.0.0.1.xip.io
+
+cert-manager:
+  webhook:
+    enabled: false
```
Comment (betatim) [inline config/turing.yaml]: Why did we turn this off?

Comment (betatim) [reply config/turing.yaml]: This is the comment that lead to it being disabled https://github.com/jupyterhub/mybinder.org-deploy/issues/1148#issuecomment-590047510

Comment (consideRatio) [reply config/turing.yaml]: https://github.com/jupyterhub/mybinder.org-deploy/issues/1148#issuecomment-590047510

Comment (sgibson91) [reply config/turing.yaml]: Yes sorry, I left this info in the commit message rather than the PR

**Pull request merged**